chain/indexer.go

```diff
@@ -253,7 +258,7 @@ func (t *TipSetIndexer) TipSet(ctx context.Context, ts *types.TipSet) error {
 
 			// If we have actor processors then find actors that have changed state
 			if len(t.actorProcessors) > 0 {
-				changes, err := t.node.StateChangedActors(tctx, parent.ParentState(), child.ParentState())
+				changes, err := t.stateChangedActors(tctx, parent.ParentState(), child.ParentState())
```

Comment:
From the tracing added in subsequent PR's I've observed this optimized call reducing the runtime of the method from ~17sec to ~3sec.

Reply:
Very nice